core/README.md

```diff
@@ -1,5 +1,5 @@
 # eolearn.core subpackage
 
-The main subpackage which implements basic building blocks (EOPatch, EOTask and EOWorkflow) and commonly used functionalities.
+The deprecated way to install `eolearn.core`. See [migration guide](https://github.com/sentinel-hub/eo-learn/issues/733) on how to correctly install the `eo-learn` package.
```

Comment:
Should we include the version here, like "how to correctly install the `eo-learn>=1.5` package.

Reply:
Unsure, seems a bit redundant :thinking:

Reply:
we are after all trying to get people to use 1.5.0 and onward